docsrc/conf.py

```diff
@@ -268,9 +269,50 @@
 
 # One entry per manual page. List of tuples
 # (source start file, name, description, authors, manual section).
-man_pages = [
+man_pages = []
+
+import glob, os, io
+
+# Which paths relate to which man page sections
+pathset = [
+    ("sasl/reference/manpages/systemcommands/", 8),
+    ("sasl/reference/manpages/library/", 3),
+    ("sasl/reference/manpages/usercommands/", 1),
+    ("sasl/reference/manpages/configs/", 5)
 ]
 
+# For each man page section/path,
+# look for all the rst files.
+# With each file, check if there's an '.. author: ' attribution (strict on
+# spacing and case sensitivity) and add to the author info.
+# Then add the file with all its details into the man_page array.
+# If the file is an :orphan:, then don't include it.
+current = os.path.abspath(os.getcwd())
+for tuple in pathset:
+    os.chdir(tuple[0])
```

Comment:
This line errors out for me because `sasl/reference/manpages/systemcommands/` doesn't exist.

I think it would be good to make it just skip that tuple if the directory doesn't exist?  That way if we later add those types of manpages, it'll already know how to find them.

I assume but haven't verified that if I remove systemcommands from pathset, then it will choke on usercommands or configs (which also don't currently exist)...